src/app/zap-templates/zcl/data-model/chip/keypad-input-cluster.xml

```diff
@@ -26,26 +26,26 @@ limitations under the License.
     <!-- TODO: Add feature map once it is available -->
     <description>This cluster provides an interface for controlling a device like a TV using action commands such as UP, DOWN, and SELECT.</description>
 
-    <command source="client" code="0x00" name="SendKey" optional="false">
+    <command source="client" code="0x00" name="SendKeyRequest" response="SendKeyResponse" optional="false">
       <description>Upon receipt, this SHALL process a keycode as input to the media device.</description>
-      <arg name="keyCode" type="KeypadInputCecKeyCode"/>
+      <arg name="keyCode" type="CecKeyCode"/>
     </command>
 
-    <command source="server" code="0x00" name="SendKeyResponse" optional="false">
+    <command source="server" code="0x01" name="SendKeyResponse" optional="false">
       <description>This command SHALL be generated in response to a SendKey Request command.</description>
-      <arg name="status" type="KeypadInputStatus"/>
+      <arg name="statusEnum" type="StatusEnum"/>
     </command>
 
   </cluster>
 
-  <enum name="KeypadInputStatus" type="ENUM8">
+  <enum name="StatusEnum" type="ENUM8">
     <cluster code="0x0509"/>
     <item name="Success" value="0x00"/>
     <item name="UnsupportedKey" value="0x01"/>
     <item name="InvalidKeyInCurrentState" value="0x02"/>
   </enum>
 
-  <enum name="KeypadInputCecKeyCode" type="ENUM8">
+  <enum name="CecKeyCode" type="ENUM8">
```

Comment:
There is no such enum in the spec, as far as I can tell.

Reply:
Yes. But CEC as a protocol has already predefined per their spec. Here's the google's implementation which I've copied:
https://android.googlesource.com/platform/frameworks/base.git/+/master/services/core/java/com/android/server/hdmi/HdmiCecKeycode.java

Reply:
Ah, I had missed the bit in the spec where it says this uint8 is in fact a CEC keycode.  Given that, OK.